load.sh

```diff
@@ -74,7 +74,7 @@ case $1 in
 
      *)
         echo "Error: Unknown option $2"
-        print_usage
+        print_usageUSER
```

Comment:
Ouch, this is wrong and :( not planned typo, and why we have to check the code. I have to delete 'USER' part of the string.